packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/EditInvoiceDetails.tsx

```diff
@@ -243,20 +244,24 @@ const EditInvoiceDetailsContent = ({
             })}
             style={{ marginBottom: theme.spacing(7) }}
           >
-            <Typography variant="h1" sx={{ mb: 7 }}>
-              {t(i18n)`Edit invoice ${
-                invoice.document_id ?? INVOICE_DOCUMENT_AUTO_ID
-              }`}
-            </Typography>
             <Stack direction="column" spacing={4}>
-              <CustomerSection disabled={isLoading} />
+              <BillToSection disabled={isLoading} />
+              <Typography
+                variant="h1"
+                sx={{ mb: 2 }}
+                data-testid={ActiveInvoiceTitleTestId.ActiveInvoiceTitleTestId}
+              >
+                {t(i18n)`Invoice`}{' '}
+                <span style={{ color: '#707070' }}>
```

Comment:
The colors shouldn't be hardcoded like that, ca we please stick to our design system approach?